openmrs/apps/clinical/formConditions.js

```diff
@@ -396,11 +396,11 @@ Bahmni.ConceptSet.FormConditions.rulesOverride = {
     "Gynecology/Obstetrics" (formName, formFieldValues, patient) {
         if (patient.gender === "F") {
             return {
-                show: ["Gynecology/Obstetrics"],
+                showForm: ["Gynecology/Obstetrics"]
```

Comment:
So I was thinking we use a flag instead of an array here since showForm should only every apply to the form in question. So something like this:
```
return {
    showForm: patient.gender === "F",
}